### DISCUSSION TITLE
Dioden - Nutzt sie jemand?

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#550] [Q&A]
Es ist keine OpenDTUonBattery Frage, da aber so viele Profis hier im Forum sind, frage ich mal.

Ich bin per Zufall auf zwei Bilder gestoßen:
 
![Dioden Schaltung](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/54c31dba-cb6c-4ea9-9bf6-ff746723685d)

Das Erste, was ich bemerkt habe: Es reicht wenn man die Dioden auf + Plus installiert (braucht nicht auch auf - minus).
 
Da ich vor habe, mein Victron 150/35 mit einer 3S2P Konfiguration zu betreiben (schon mit [Victron MPPT Calculator](https://mppt.victronenergy.com#djU7U0NDMDIwMDM1MDAwO2V5SnplWE4wWlcwaU9qUTRMQ0ppWVhSMFpYSjVJam8xTkgwPTtleUpzWVhRaU9qVXdMamN6TnpRekxDSnNibWNpT2pjdU1EazRNakEyT0N3aWJtRnRaU0k2SWtKdmJtNHNJRWRsY20xaGJua2lmUT09O2V5Sm1aV0YwZFhKbGN5STZXMTBzSW5SbGNtMXBibUZzSWpwMGNuVmxMQ0poYkd4dmQwOTJaWEp6YVhwbFpDSTZkSEoxWlgwPTtXM3NpYlc5a2RXeGxJanA3SW5SNWNHVWlPaUpqZFhOMGIyMGlMQ0poY25ScFkyeGxUblZ0WW1WeUlqcHVkV3hzTENKcFRYQndJam94TVM0NE5Dd2lhVlJsYlhCRGIyVm1aaUk2TUM0d05EZ3NJbWx6WXlJNk1USXVOREVzSW0xaGJuVm1ZV04wZFhKbGNpSTZJaUlzSW5CdmQyVnlJam8wTVRBc0luWk5jSEFpT2pNMExqWXNJblpVWlcxd1EyOWxabVlpT2kwd0xqSTNMQ0oyYjJNaU9qUXhMallzSW1sa0lqcHVkV3hzZlN3aVkzVnpkRzl0VFc5a2RXeGxJanB1ZFd4c0xDSnpaWEpwWlhNaU9qTXNJbkJoY21Gc2JHVnNJam95TENKaVlYUjBaWEo1Um14dllYUldiMngwWVdkbElqbzFOQ3dpYldsdVZHVnRjR1Z5WVhSMWNtVWlPaTB5TUN3aWJXRjRWR1Z0Y0dWeVlYUjFjbVVpT2pZMUxDSjJiMngwWVdkbElqb3lOQ3dpWTJGaWJHVk1aVzVuZEdnaU9qRTFMQ0pqWVdKc1pVTnliM056VTJWamRHbHZiaUk2Tm4xZA==) berechnet und es ist OK), bin ich plötzlich auf dem Geschmack gekommen, dank Dioden, könnte ich sogar die Strings in verschieden Richtungen platzieren (ein String auf der Vorderseite des Hauses und eins auf der Hinterseite).

Hat jemand hier Erfahrungen damit gemacht?

Ich habe schon verstanden, es gibt die billigen Dioden fuer 10EUR das Paar (und brennen schnell) 
![BilligDioden](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/0e6d81b4-3674-4945-973d-d4c6b745f45b)
 
![Verschmorte 20A  Dioden 01](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/57097bd8-ab71-4e08-8046-5d32f9b55a4d)

![Verschmorte 20A  Dioden 02](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/83a331e4-53cb-486f-8278-a5c32982e829)

und die besseren, mit Kühlung, die deutlich teurer sind (12-20EUR p. Stueck):
![Diode bis 55A](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/e3473995-456f-426b-8a39-e66ae629ca8f)

**Comment by JaMo523** [comment]
Suche mal nach einem Datenblatt/Whitepaper von deinen Modulen. Bei meinen Modulen sind Bypass Dioden verbaut.

**Reply by Manos1966** [reply]
@JaMo523 Ja, meine haben sie auch.

Ist es sinnvoll, einen ganzen String durch die Diode des ersten Moduls des anderen Strings zu stoppen?

**Reply by JaMo523** [reply]
Du willst darauf hinaus, das du zwei Strings mit verschiedener Ausrichtung hast und das je nachdem wo die Sonne steht nur noch Strom durch den einen String fließt?

**Reply by Manos1966** [reply]
Liest du nicht den Text, der vorher geschrieben ist? 😉 

> Da ich vor habe, mein Victron 150/35 mit einer 3S2P Konfiguration zu betreiben (schon mit Victron MPPT Calculator berechnet und es ist OK), bin ich plötzlich auf dem Geschmack gekommen, **dank Dioden, könnte ich sogar die Strings in verschieden Richtungen platzieren (ein String auf der Vorderseite des Hauses und eins auf der Hinterseite).**

**Reply by spcqike** [reply]
sind anti-reverse-dioden nicht was anderes, als bypass Dioden?

Die Bypass Diode soll doch ein Modul "Überbrücken", damit dieses nicht den gesamten String herunterzieht. die Anti-reverse-Diode soll verhinden, dass der Strom nicht "Rückwärts" durch den gesamten String fließt und damit mindestens Verluste entstehen (Solarpaneele können auch wie eine LED "[leuchten](https://youtu.be/o4WFACkiV-8?si=biyLqYIisv-pEOHK&t=424)", wenn auch im nicht sichtbaren IR Spektrum), wenn nicht sogar die Paneele beschädigen, oder? 

ich mich damit auch noch nicht näher beschäftigt, da man das bei lediglich 2 Paneelen an einem HM-600 einfach nicht braucht :D

**Reply by JaMo523** [reply]
oben im Bild ist parallel die Bypass Dioden zusehen und in Reihe die anti-reverse Diode. Die verhalten sich so wie du beschrieben hast.

**Reply by Manos1966** [reply]
Habe ich auch so gelesen (anti-reverse-dioden vs bypass-dioden) @spcqike bin ich aber zum Entschluss gekommen, es handelt sich um eine Beschreibung der Wirkung. 
Ich habe nirgendwo anti-reverse oder bypass  in der Produktbeschreibung gefunden.

Ich war schon sehr Überrascht als ich den Einsatz der Dioden am ersten Bild (Bild 18) gesehen habe!
Beim Preis von 15EUR pro (anständige) Diode ist es der wirtschaftlicher Sinn natürlich fragwürdig, aber das lasse ich erstmals aussen vor.

> da man das bei lediglich 2 Paneelen an einem HM-600 einfach nicht braucht :D

Oh spcqike 🙄  Da gibt's aber Luft nach oben 🤣 🤣 🤣

**Reply by JaMo523** [reply]
Ich habe mir schon den Text durchgelesen. 

Ganz vereinfacht erklärt:

Gehen wir davon aus, dass die beiden Strings nicht mit einander verbunden sind:

String mit voller Sonne:
Dein MPPT-Tracker wird den optimalen Leitungspunkt suchen. Durch die Belastung bricht dann die Modulspannung ein und wird unterhalb der Leerlaufspannung(unbelastet) sein. 

String mit Schatten:
Leerlaufspannung (unbelastet).

Jetzt verbinden wir die beiden Strings.
In der Parallelschaltung ist die Spannung über beide Strings gleich hoch.

Also bricht die Spannung von den unbelasteten Modulen auch ein und ein kleiner Strom fließt vorwärts Richtung MPPT-Tracker.

Es wird also wenn dein MPPT-Tracker arbeitet selten ein Rückwärtsstrom geben und wenn dann ist der nicht groß. Er könnte theoretisch so groß sein wie der, der durch den anderen String produziert wird. Falls dieser Auftritt wird er bei zwei Strings nicht zu einem Schaden führen.

Mit den Dioden kannst du den Rückwärtsstrom unterbinden. Es bringt dir aber keinerlei Vorteile. Du hättest den Nachteil, dass dauerhaft Leistung an den Dioden abfällt.

**Reply by Manos1966** [reply]
Danke @JaMo523 das klingt gut!
